keps/sig-node/2570-memory-qos/README.md

```diff
@@ -348,12 +387,86 @@ type LinuxContainerResources struct {
 ```
 
 ### Test Plan
+
+<!--
+**Note:** *Not required until targeted at a release.*
+The goal is to ensure that we don't accept enhancements with inadequate testing.
+
+All code is expected to have adequate tests (eventually with coverage
+expectations). Please adhere to the [Kubernetes testing guidelines][testing-guidelines]
+when drafting this test plan.
+
+[testing-guidelines]: https://git.k8s.io/community/contributors/devel/sig-testing/testing.md
+-->
+
+[X] I/we understand the owners of the involved components may require updates to
+existing tests to make this code solid enough prior to committing the changes necessary
+to implement this enhancement.
+
+Overall Test plan:
+
 For `Alpha`, unit tests were added to test functionality for container/pod/node level cgroup with containerd and CRI-O.
 
-For `Beta`, e2e-node tests were added, tests were added for dockershim
+For `Beta`, e2e-node tests were added, tests were added for dockershim.
```

Comment:
why dockershim?

Reply:
Good catch, this was part of KEP prior to containerd deprecation. I updated to include the new plan for alpha in 1.27 of adding E2E node tests to validate that the MemoryQoS settings are applied correctly.